### ISSUE TITLE
Mutiple calls 

### ISSUE THREAD

**Issue by rafaeldfdias** [bsgriggs/pagination#4]
Hello,
I don't know if this due to the way that I assembled the page, but when I enter a page which has both the Advanced Pagination and Advanced Sorting, I get three calls to the service that I am consuming. Is this normal, or is  this a widget limitation?
This is the way that I have my page assembled
![image](https://github.com/user-attachments/assets/4027c5c4-c146-434a-9c23-20259315f9e4)

**Comment by bsgriggs** [comment]
Hi there, can you please provide screenshots of the advanced pagination & advanced sorting widget's settings?

**Comment by rafaeldfdias** [comment]
Hi, 
sure!
Here they are:
Sorting:
![image](https://github.com/user-attachments/assets/65575ee7-c023-46a7-b00f-be49daa60019)
Pagination:
![image](https://github.com/user-attachments/assets/3308e452-6336-4002-b1cd-7d6ab9b94717)
And just to give you a little bit of more context, this is the page structure with the helper object that gets refreshed:
![image](https://github.com/user-attachments/assets/ced72058-9270-4ff4-a8fd-25248f1c138d)

best regards

**Comment by bsgriggs** [comment]
Make sure that your UIHelper's default value for Page is set to 1. It may be refreshing due to the Auto Correct setting and the Page Number being invalid.

**Comment by rafaeldfdias** [comment]
Hi @bsgriggs thanks for your help. 
I took another look at the project and I found out why this was happening. It was related with the fact that I had left behind some old filters in the Grid wide filters tab of the datagrid2. (Section shown below)
![image](https://github.com/user-attachments/assets/7748057a-9d4c-4787-b300-250966445525)
 Once I removed them, it stopped with the duplicate request.
